### PR TITLE
fix: Admin panel member since changed to date

### LINKED ISSUE #5542: Admin Pages: Change "Member Since" to date
The column "member since" shows x days ago instead of a date. Please change the column to show a date on https://eventyay.com/admin/users/

Screenshot from 2020-11-09 18-01-41

### PR DESCRIPTION
**Pull request by codedsun**
Fixes #5542  fix: Admin panel member since changed to date

Changes:
image

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by codedsun** [comment]
@iamareebjamal - Review!

**Review by mariobehling — CHANGES_REQUESTED** [review]
Please change 24 hour format + AM/PM to 12 hour format + AM/PM

**Comment by iamareebjamal** [comment]
Please use the format decided in #5533

**Comment by codedsun** [comment]
image

Done @mariobehling @iamareebjamal

**Comment by iamareebjamal** [comment]
Is that the format mentioned here? https://github.com/fossasia/open-event-frontend/pull/5533#pullrequestreview-526527704

**Comment by codedsun** [comment]
@iamareebjamal - Now is it correct?

image


Also IST should be harcoded?

**Comment by iamareebjamal** [comment]
No need of IST

File: open-event-frontend.iml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="WEB_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
```
Comment (iamareebjamal) [inline open-event-frontend.iml:9]: remove

**Comment by codedsun** [comment]
Done @iamareebjamal

**Comment by mariobehling** [comment]
No trailing 0 please. Use 7 Nov, instead of 07 Nov.

**Comment by mariobehling** [comment]
Use 4:00 PM instead of 04:00 PM.

**Comment by codedsun** [comment]
@mariobehling

**Comment by codedsun** [comment]
@iamareebjamal Can this be merged?

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**